packages/e2e-test-utils/src/user.js

```diff
@@ -45,7 +45,7 @@ export async function setCurrentUser(username, password) {
     () => document.querySelector('.display-name')?.textContent || ''
```

Comment:
`expect(arg).toMatchTextContent()` requires `arg` to be an ElementHandle. This `page.evaluate` will resolve to a `string`. I think we should use [page.$](https://pptr.dev/api/puppeteer.page._) here.
`const currentUser = await page.$('.display-name')`